src/backend/base/langflow/components/data/file.py

```diff
@@ -49,6 +46,13 @@ class FileComponent(Component):
             advanced=True,
             info="If true, parallel processing will be enabled for zip files.",
         ),
+        StrInput(
```

Comment:
Why not IntInput here?

Reply:
Hah... mostly because i forgot Intinput exists :) 

I'll update now.